vendors/twtg/profiles/twtg_RFGroup1_1.0.4a_RP2-1.0.3_classA.yaml

```diff
@@ -0,0 +1,98 @@
+        maxNbTrans:
+        # How many uplink transmissions does the device use in Confirmed mode if it doesn't receive any DL ACK?
+        maxRedundancyForConfirmedUL: 0
+
+# The following section defines the default settings used by the device during iniial boot stage. Leave empty if your device respects the 
+# default values indicated by LoRaWAN Regional Parameters (PHY) specification. Personalize fields only when different from default PHY values.
+bootSettings: 
+    # Delay between the end of uplink transmission and the start of downlink RX1 slot, in milliseconds. Refer to 'receiveDelay1' in LoRaWAN 
+    # Specification.
+    rx1Delay: 1000
+    # RX1 data rate offset, defining the offset between uplink data rate and the corresponding downlink data rate used for RX1 slot.
+    rx1DROffset:
+    # Rx2 DataRate, used to send DL packets over RX2 slot. Note: Data Rate encoding is region-specific, see the applicable LoRaWAN Regional 
+    # Profile.
+    rx2DataRate:
+    # Rx2 Frequency expressed in MHz, used to send DL packets over RX2 slot.
+    rx2Frequency:
+    # Default data rate used for Class B ping slots. Note: Data Rate encoding is region-specific, see the applicable LoRaWAN Regional Profile.
+    pingSlotChannelDataRate:
+    # Default frequency (in MHz) used for Class B ping slots.
+    pingSlotChannelFrequency:
+    # beaconFrequency Boot value used by the Device to set the beacon broadcast frequency. This frequency is then aligned by the Network Server to the target value set in RF region.
+    beaconFrequency:
+    # Default downlink dwell time (valid only for AS923 and AU915 profiles). Possible values 0 (no limit) or 1 (400ms).
+    downlinkDwellTime: 0
+    # Default uplink dwell time (valid only for AS923 and AU915 profiles). Possible values 0 (no limit) or 1 (400ms).
+    uplinkDwellTime: 0
+
+# Which MAC commands are supported by your device and which are not? Leave it as the template if your device supports all the MAC commands specified for your regional
+# profile. Put 'true' only if a command is supported.
+supportedMACCommands:
+    linkADRReqAns: true
+    devStatusReqAns: true
+    joinRequestAccept: true
+    dutyCycleReqAns: true
+    linkCheckReqAns: true
+    rxParamSetupReqAns: true
+    rxTimingSetupReqAns: true
+    newChannelReqAns: true
+    dlChannelReqAns: true
+    txParamSetupReqAns: false
+    pingSlotChannelReqAns: false
+    pingSlotInfoReqAns: true
+    deviceTimeReqAns: true
+    beaconFreqReqAns: false
+
+# Does your device support [Technical Recommendation for LoRaWAN L2 1.0.x Join Security] recommending implemeting the DevNonce as a counter for 
+# OTAA devices? See https://lora-alliance.org/resource_hub/technical-recommendations-for-preventing-state-synchronization-issues-around-lorawan-1-0-x-join-procedure/
+# Possible values: true, false. This field is automatically forced to true for LoRaWAN MAC versions 1.0.4 and 1.1.
+devNonceCounterBased: true
```

Comment:
As this profile is for ` loRaWANMacVersion: 1.0.4`, can we just remove this line?

Reply:
It's best to keep it for clarity, if you don't mind.